--- index.jsp.original	2015-09-03 11:08:40.000000000 -0300
+++ index.jsp	2015-09-03 16:47:46.000000000 -0300
@@ -1,5 +1,11 @@
 <html>
 <body>
 <h2>Hello World!</h2>
+<%
+String portOffset = System.getProperty("jboss.socket.binding.port-offset", "0");
+int port = 9999 + Integer.valueOf(portOffset);
+com.example.Main.main(new String[]{"localhost", String.valueOf(port), "beltrano", "outrasenha"});
+out.println("<h3>" + com.example.Main.getResult() + "</h3>");
+%>
 </body>
 </html>
